ha/monitor/k8s/object_monitor.py

```diff
@@ -148,12 +151,12 @@ def run(self):
                     break
 
                 Log.debug(f"Received event {an_event}")
-                alert = EventParser.parse(self._object, an_event, self._object_state)
+                alert, event = EventParser.parse(self._object, an_event, self._object_state)
                 if alert is None:
                     continue
                 if self._starting_up:
                     if an_event[K8SEventsConst.TYPE] == EventStates.ADDED:
-                        alert.is_status = True
+                        alert = event.update_event(key_to_update={"is_status": True})
```

Comment:
Please make sure this method is provided by this class.

Reply:
Sure. Yes.